### DISCUSSION TITLE
how to use escape character in pkl project

### DISCUSSION THREAD

**Discussion by ruby109** [apple/pkl#381] [Q&A] [answered]
I'm using pkl to generate xcodegen's project.yml, and there is a key call "UISupportedInterfaceOrientations~ipad".
![CleanShot 2024-04-01 at 17 15 34@2x](https://github.com/apple/pkl/assets/18363005/746123f9-d16c-4e81-83de-abb0bd2299af)

I would like to make a object like this :
``` pkl
UISupportedInterfaceOrientations~ipad {
    "UIInterfaceOrientationPortrait"
    "UIInterfaceOrientationPortraitUpsideDown"
}
```
But this not work, have a error log:
```
Reason: –– Pkl Error ––
Token recognition error at: `~i`

108 | UISupportedInterfaceOrientations~ipad {
                                      ^
at project (file:///Users/***/***-ios/project.pkl, line 108)
Domain: scripts
Time: 22:42:08.733
```
How can I solve this problem?

**Comment by StefMa** [comment, marked as answer]
Probably with backticks:
```
`UISupportedInterfaceOrientations~ipad`
```

**Reply by ruby109** [reply]
Thank you very much!
